modules/enabling-multi-cluster-console.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * assemblies/web-console.adoc
+
+:_content-type: PROCEDURE
+[id="enable-multi-cluster-console_{context}"]
+= Enabling multicluster in the web console
+
+:FeatureName: Multicluster console
+include::snippets/technology-preview.adoc[leveloffset=+1]
+//
+
+.Prerequisites
+* Your cluster must be using the latest version of {product-title}.
+* You must have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/install/index[Red Hat Advanced Cluster Management (ACM) for Kubernetes 2.5] installed.
+* You must have administrator privileges.
+
+[WARNING]
+====
+Do not set this feature gate on production clusters. You will not be able to upgrade your cluster after applying the feature gate, and it cannot be undone.
+====
+
+.Procedure
+
+. Log in to the {product-title} web console using your credentials.
+
+. Enable ACM in the administrator perspective by navigating from *Admin* -> *ClusterSettings* -> *Configuration* -> *Console Details* -> *Console Plugins* and click *enable*.
```

Comment:
![Screen Shot 2022-06-08 at 13 30 07](https://user-images.githubusercontent.com/12692381/172539110-89a348c0-d063-4f02-a30b-5b91406ecd07.png)
IIRC we need enable both `acm` and `mce` consoleplugin to have multicluster feature, also some slight changes needed
- `Admin` should be updated to `Administration`
- `ClusterSettings` should be updated to `Cluster Settings`(there is space)
- `Console Details` should be updated to `Console details`(lower case)
- also we need to emphasize that we should go to console.operator.openshift.io details page, not  console.config.openshift.io details page, we have two console entries on Configuration page
-

Reply:
shall we add more words describing how to enable the plugin instead of just say 'enable? 
attaching a screenshot for better understanding
![Screen Shot 2022-06-08 at 13 42 11](https://user-images.githubusercontent.com/12692381/172540563-a75e4bd4-2395-40b6-9739-46c1cd3cf6ce.png)

Reply:
IIRC we need enable both `acm` and `mce` consoleplugin to have multicluster feature?